cmd/kured/main.go

```diff
@@ -308,7 +311,12 @@ func acquire(lock *daemonsetlock.DaemonSetLock, metadata interface{}, TTL time.D
 	}
 }
 
-func release(lock *daemonsetlock.DaemonSetLock) {
+func release(lock *daemonsetlock.DaemonSetLock, releaseDelay time.Duration) {
+	if releaseDelay > 0 {
+		log.Infof("Delaying lock release by %v", releaseDelay)
+		time.Sleep(releaseDelay)
```

Comment:
I am not too fond of this approach, as it's not very clear that it's throttling the whole process at first sight.

However, I am not sure how to refactor this to make it more obvious.

Reply:
Are you concerned about the usability of the tool or the readability of the code in this case?

Reply:
The latter. Sustainability of the code.